MyHashSet.java

```diff
@@ -0,0 +1,70 @@
+// Time Complexity : O(1) to add, udpdate
```

Comment:
If all elements are stored in a single bucket, the time complexity here will be O(n).